src/backend/HostedFileStorageSystemClient.js

```diff
@@ -10,6 +10,14 @@ const EMPTY_SOURCE_CONTENT = {
   content: [{ type: "paragraph", attrs: { tags: {} } }],
 };
 
+function getErrorMessage(response, defaultMsg) {
+  if (response.status === 401) {
```

Comment:
can u add a comment saying that this status will only be triggered for google oauth errors